src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -169,22 +174,28 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
                 && (latestStableVersion.version == null || latestPrerelease.version > latestStableVersion.version) &&
                 !latestPrerelease.version.Equals(installedVersion))
             {
-                _versions.Add(new DisplayVersion(latestPrerelease.version, Resources.Version_LatestPrerelease, isDeprecated: latestPrerelease.isDeprecated));
+                VersionRange latestPrereleaseVersionRange = VersionRange.Parse(latestPrerelease.version.ToString(), allowFloating: false);
+                _versions.Add(new DisplayVersion(latestPrereleaseVersionRange, latestPrerelease.version, Resources.Version_LatestPrerelease, isDeprecated: latestPrerelease.isDeprecated));
             }
 
             // Add latest stable if needed
             if (latestStableVersion.version != null &&
                 !latestStableVersion.version.Equals(installedVersion))
             {
-                _versions.Add(new DisplayVersion(latestStableVersion.version, Resources.Version_LatestStable, isDeprecated: latestStableVersion.isDeprecated));
+                VersionRange latestStableVersionRange = VersionRange.Parse(latestStableVersion.version.ToString(), allowFloating: false);
+                _versions.Add(new DisplayVersion(latestStableVersionRange, latestStableVersion.version, Resources.Version_LatestStable, isDeprecated: latestStableVersion.isDeprecated));
             }
 
+            // Only the current installed version is displayed, we update the separator so its not removed from the list when filtering.
+            IsBeforeNullSeparator = _versions.Count == 1;
```

Comment:
Is this property still being used? I can't find it anywhere. Did check `ProjectView` on your branch...

Reply:
Yes im still using this property for filtering, I use it for the case `*` because * will show every version, but we dont want to show the Installed and Latest version that are before the null separator.

![image](https://user-images.githubusercontent.com/43253759/150861035-5a469e8f-de4d-475a-aa58-5712a273339f.png)
![image](https://user-images.githubusercontent.com/43253759/150861071-5fbd8beb-4c96-43f5-b2a3-d777e27ff91d.png)